### PR TITLE
lastpass lookup: use config manager, improve documentation

### PR DESCRIPTION
**Pull request by lungj**
##### SUMMARY
Cleaned up LastPass lookup plug-in to meet coding standards.

Since the issues in #5012 are a result of me previously basing code off of this, by making this adhere to current standards for this repository, hopefully this will result in less work for reviewers in the future.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
`lastpass`

### CODE REVIEW

File: plugins/lookup/lastpass.py [hunk 1]
```diff
@@ -11,21 +11,24 @@
       - Andrew Zenk (!UNKNOWN) <azenk@umn.edu>
     requirements:
       - lpass (command line utility)
-      - must have already logged into lastpass
-    short_description: fetch data from lastpass
+      - must have already logged into LastPass
+    short_description: fetch data from LastPass
     description:
-      - use the lpass command line utility to fetch specific fields from lastpass
+      - use the lpass command line utility to fetch specific fields from LastPass
```
Comment (felixfontein) [inline plugins/lookup/lastpass.py:17]:
```suggestion
      - Use the lpass command line utility to fetch specific fields from LastPass.
```

File: plugins/lookup/lastpass.py [hunk 2]
```diff
@@ -11,21 +11,24 @@
...
     options:
       _terms:
         description: key from which you want to retrieve the field
```
Comment (felixfontein) [inline plugins/lookup/lastpass.py:20]:
```suggestion
        description: Key from which you want to retrieve the field.
```

File: plugins/lookup/lastpass.py [hunk 3]
```diff
@@ -11,21 +11,24 @@
...
-        required: True
+        required: true
+        type: list
+        elements: str
       field:
-        description: field to return from lastpass
+        description: field to return from LastPass
```
Comment (felixfontein) [inline plugins/lookup/lastpass.py:25]:
```suggestion
        description: Field to return from LastPass.
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution. Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by felixfontein** [comment]
What's missing are similar changes to the unit tests as in https://github.com/ansible-collections/community.general/pull/5012#discussion_r932604228.

File: changelogs/fragments/5022-lastpass-lookup-cleanup.yml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+trivial:
+ - lastpass - use config manager API in tests; improve documentation.
```
Comment (felixfontein) [inline changelogs/fragments/5022-lastpass-lookup-cleanup.yml:2]: Test and documentation changes do not need a changelog fragment. But the main change of this PR is something that must be mentioned:
```suggestion
minor_changes:
 - lastpass - use config manager for handling plugin options (https://github.com/ansible-collections/community.general/pull/5022).
```

**Review by felixfontein — APPROVED** [review]
Looks good to me. If nobody complains, I'll merge this for the next release.

**Comment by lungj** [comment]
> Looks good to me. If nobody complains, I'll merge this for the next release.

I left this as a draft in case there are any new issues identified in #5012 that are also in this plug-in.

**Comment by felixfontein** [comment]
I'd prefer to merge this for 5.4.0 no matter whether #5012 is merged by then or not. The most important change here has been made (use of `set_options()` / `get_option()`).

**Comment by felixfontein** [comment]
@lungj thanks for fixing this!

**Pull request merged**